build.gradle

```diff
@@ -262,7 +262,7 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'logging-spring', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'logging-httpcomponents', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
-  compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '4.2.36'
+  compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '4.2.27'
```

Comment:
This is tool old, try to roll forward with your sscs-common version in https://github.com/hmcts/sscs-common/pull/783

Reply:
Updated